lib/network/modules/EdgesHandler.js

```diff
@@ -38,6 +38,7 @@ class EdgesHandler {
       dashes: false,
       font: {
         color: '#343434',
+        highlight: '#ff0000',
```

Comment:
I would consider this a breaking change because edge labels get now colored in red if highlighted. I think we could live with something like `#000000`but not red at this point.